src/main/java/edu/cornell/kfs/pmw/batch/businessobject/PaymentWorksVendor.java

```diff
@@ -1233,20 +1233,20 @@ public void setNewYorkDiversityCertificates(String newYorkDiversityCertificates)
         this.newYorkDiversityCertificates = newYorkDiversityCertificates;
     }
 
-    public String getFederalDivsersityClassifications() {
-        return federalDivsersityClassifications;
+    public String getFederalDiversityClassifications() {
+        return federalDiversityClassifications;
     }
 
-    public void setFederalDivsersityClassifications(String federalDivsersityClassifications) {
-        this.federalDivsersityClassifications = federalDivsersityClassifications;
+    public void setFederalDiversityClassifications(String federalDivsersityClassifications) {
```

Comment:
The local variable still needs the typo fix.

Reply:
ok